CMakeLists.txt

```diff
@@ -187,13 +188,12 @@ macro(spirv_cross_add_library name config_name library_type)
 
 	if (NOT SPIRV_CROSS_SKIP_INSTALL)
 		install(TARGETS ${name}
-			EXPORT ${config_name}Config
+			EXPORT SPIRV-CrossTargets
```

Comment:
Wouldn't this break existing consumers of SPIRV-Cross?

Reply:
Or you can let the new `SPIRV-CrossConfig.cmake` make use components:
```cmake
@PACKAGE_INIT@
find_package(spirv_cross_core QUIET)
set(SPIRV-Cross_core_FOUND ${spirv_cross_core_FOUND})

find_package(spirv_cross_glsl QUIET)
set(SPIRV-Cross_core_FOUND ${spirv_cross_glsl_FOUND})

find_package(spirv_cross_hlsl QUIET)
set(SPIRV-Cross_core_FOUND ${spirv_cross_hlsl_FOUND})

find_package(spirv_cross_msl QUIET)
set(SPIRV-Cross_core_FOUND ${spirv_cross_msl_FOUND})

find_package(spirv_cross_cpp QUIET)
set(SPIRV-Cross_core_FOUND ${spirv_cross_cpp_FOUND})

find_package(spirv_cross_reflect QUIET)
set(SPIRV-Cross_core_FOUND ${spirv_cross_reflect_FOUND})

find_package(spirv_cross_c QUIET)
set(SPIRV-Cross_core_FOUND ${spirv_cross_c_FOUND})

find_package(spirv_cross_c_shared QUIET)
set(SPIRV-Cross_core_FOUND ${spirv_cross_c_found_FOUND})

check_required_components(SPIRV-Cross)
```

Then you can do things like:
```cmake
find_package(SPIRV-Cross REQUIRED COMPONENTS spirv_cross_c_shared)
```
while still remaining compatible with
```cmake
find_package(spirv_cross_c_shared REQUIRED)
```

(I think CMake does not allow exporting targets to more then one export set)